windows/getopt.h

```diff
@@ -1,4 +1,5 @@
 #ifndef __GETOPT_H__
+/** This file has been modified by D. Lemire for use in simdjson. **/
```

Comment:
Will be happy when we get it all in cxxopts :) Thanks for leaving a changelog in the file!

Reply:
Right now cxxopts is not available when git is unavailable and it is not reasonable to deactivate our tests if networking and git are unavailable.

So if we move everything to cxxopts, I think we need to bring a hard copy of cxxopts in the project. Yeah. I know it is unfortunate, but at some point, we need to build on an isolated machine.

Reply:
Yeah. It's not terrible as long as we give ourselves a way to check for and update our dependencies (the real reason for a submodule).